File: _debug2.js
const fs = require('fs');
const h = fs.readFileSync('C:/Users/wzgui/.openclaw-autoclaw/workspace/sportpredict/index.html', 'utf8');
const U = String.fromCharCode(65533);

// Find first corruption
const idx = h.indexOf('closeWatchModal()');
if (idx >= 0) {
  const seg = h.substring(idx, idx + 60);
  console.log('Segment:', JSON.stringify(seg));
  for (let i = 0; i < seg.length; i++) {
    if (seg.charCodeAt(i) > 127 || seg[i] === '?') {
      console.log(`  pos ${i}: char=${seg[i]} code=${seg.charCodeAt(i).toString(16)}`);
    }
  }
}

File: debug2.py
filepath = r'C:\Users\wzgui\.openclaw-autoclaw\workspace\sportpredict\index.html'

with open(filepath, 'rb') as f:
    raw = f.read()
content = raw.decode('utf-8', errors='replace')

lines = content.split('\n')

# Print lines around 1625 (0-indexed: 1624)
for i in range(1623, 1635):
    print(f"L{i+1}: {repr(lines[i][:150])}")
